### ISSUE TITLE
rollup_plugin_dotenv_1.default is not a function

### ISSUE THREAD

**Issue by chrisbraddock** [Andarist/rollup-plugin-dotenv#1]
Hi - thanks for this. I'd love to get DotEnv working for us.

I'm getting this error when I try to include your plugin:

```
rollup_plugin_dotenv_1.default is not a function TypeError: rollup_plugin_dotenv_1.default is not a function
           at Object.<anonymous> (/stencil-module-issue/stencil.config.ts:16:39) at
           Module._compile (module.js:652:30) at Object.require.extensions..js
           (/Users/e071111/Documents/repos/stencil-module-issue/node_modules/@stencil/core/dist/sys/node/index.js:766:20
```

Any ideas? I'm new to Rollup ... I'm wondering if I need to add some module loader plugins as well.

There is a minimum example here: https://github.com/chrisbraddock/stencil-module-issue/tree/feature/dotenv

**Comment by Andarist** [comment]
Im not a TS user, but I suspect you (or stencil) were missing `esModuleInterop: true` in TS config. I've resolved the issue on my side to avoid this altogether and released v0.2.0. Let me know if it works for you. Thanks for taking an interest in the plugin!

**Comment by chrisbraddock** [comment]
Wohoo! It works. :) Thanks much!
